website/integrations/services/slack/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Slack
+---
+
+<span class="badge badge--secondary">Support level: authentik</span>
+
+## What is Slack
+
+> Slack is a platform for collaboration, with chat and real-time video capabilities.
+>
+> -- https://slack.com
+
+## Preparation
+
+The following placeholders will be used:
+
+-   `slack.company` is the FQDN of the Slack install.
+-   `authentik.company` is the FQDN of the authentik install.
+
+## authentik configuration
+
+### Step 1. Create custom property mappings
+
+Your Slack integration requires property mappings for `User.Email` and `User.Username` so that authetnik can retrieve and map these values from Slack.
```

Comment:
```suggestion
Your Slack integration requires property mappings for `User.Email` and `User.Username` so that authentik can retrieve and map these values from Slack.
```